src/webgpu/api/operation/command_buffer/image_copy.spec.ts

```diff
@@ -1500,6 +1547,10 @@ function* generateTestTextureSizes({
   if (modifyWidth && modifyHeight) {
     yield [modifiedWidth, modifiedHeight, textureSize[2]];
   }
+
+  if (dimension === '3d') {
+    yield [textureSize[0], textureSize[1], textureSize[2] + 1];
+  }
```

Comment:
not clear to me where we're only testing [w, h, d+1]. I guess that given the function comment, 3D here might not really matter much since the purpose of this generator is to generate differences in width+height ?

Reply:
We're also testing `[w, h, d]`, I thought it would be some extra coverage for partial-depth copies to 3D texture mip levels. WDYT?

Updated the description accordingly.

Reply:
ah okay, sounds good